deploy-board/deploy_board/templates/configs/env_config.tmpl

```diff
@@ -249,16 +249,29 @@
                         Stage Type (Immutable once set off the DEFAULT type)
                     </label>
                     <div class ="col-xs-4">
-                        <select class="form-control" name="stageType">
-                            {% get_stage_types as stageTypes %}
-                            {% for stageType in stageTypes %}
-                            {% if env.stageType == stageType %}
-                            <option value="{{ stageType }}" selected="selected">{{ stageType }}</option>
-                            {% else %}
-                            <option value="{{ stageType }}">{{ stageType }}</option>
-                            {% endif %}>
-                            {% endfor %}
-                        </select>
+                        <div class="input-group">
+                            <select class="form-control" name="stageType">
+                                {% get_stage_types as stageTypes %}
+                                {% for stageType in stageTypes %}
+                                {% if env.stageType == stageType %}
+                                <option value="{{ stageType }}" selected="selected">{{ stageType }}</option>
+                                {% else %}
+                                <option value="{{ stageType }}">{{ stageType }}</option>
+                                {% endif %}>
+                                {% endfor %}
+                            </select>
+                            <span class="input-group-btn">
+                                <button class="deployToolTip btn stage-type-btn-default" id="stageTypeBtnId" type="button" data-toggle="tooltip" title="click to see more information">
+                                    <span class="glyphicon glyphicon-question-sign"></span>
+                                </button>
+                            </span>
+                        </div>
+                        <div class="col-xs-12" id="stageTypeHelpInfo" hidden="true">
+                            LATEST - Deploys the latest available build</br>
+                            CANARY - Deploy the build after TESTING</br>
```

Comment:
After TESTING? What is TESTING?

Reply:
let me change this to 'tests' or 'testing' to make it more clear

Reply:
thank you